connectors/aws/aws-sagemaker/README.md

```diff
@@ -0,0 +1,3 @@
+# AWS SageMaker Connector
+
+The **AWS SageMaker Connector** allows you to interact with the AWS SageMaker runtime API from BPMN.
```

Comment:
Thank you. I personally find these docs as an overkill. By glancing several of those, I see they don't represent actual state of connector, since people basically, forget updating those.

IMO, short description should be sufficient unless users raise a concern they need those.